README.md

```diff
@@ -1,47 +1,75 @@
-# conu
-__CON__ -tainers __U__ -tils
-
-It provides low level __API__ for docker image/container testing.
-It is based on bash commands because primary target is to __test it
-as customers will__ use it.
-That's why it does not use python docker bindings.
-It provides python classes for:
-- __docker__ module
-  - __Image__ - image handling
-  - __Container__ - docker container handling
-- __utils__ module
-  - __Volume__ - it is mktemd utility on steroids
-  - __Probe__ - provides, simple port/file checks and waiting for this events
+# CONU
+
+Conu (CONtainer Utils) provides low level __API__ for image/container testing.
+TODO - find better punchline :)
 
 # Installation
 
-## Prerequisites
-- system packages providing binaries: `docker`, `setfacl`, `chmod`, `chcon`
-```
-$ dnf -y install acl coreutils docker
-```
-- python modules inside: No specific requirements now
+You can try out CONU in your local machine by running:
 
-## From source Code
-- Commands in git directory:
-```
-$ sudo make install
+```commandline
+git clone https://github.com/fedora-modularity/conu
+sh -c conu/requirements.sh
+pip3 install --user ./conu
 ```
 
-## COPR repo
-- Enable copr repo
-```
-dnf -y copr enable __MISSING__
-```
-- Install package
-```
-dnf -y install conu
-```
+More info [here](http://conu.readthedocs.io/en/latest/installation.html).
 
-# How to use it
+# List of features
+## Image
+- load, pull, mount and remove
+- get metadata
+- check presence of files and directories
+- copy file or directory to and from from image
+- read files in image
+- get selinux context of files in image
+- extend image using s2i
+
+## Container
+- kill, logs, exec, mount, remove, start, stop, wait, run - via api, or via binary
+- get metadata
+- shortcut methods for getting:
+    - IPv4 and IPv6 address
+    - PID of root process in container
+    - port mappings
+    - container status
+- HTTP requests support
+- open TCP connection
+- checks
+    - is running
+    - mapped port opened
+
+## Utilities
+- create and delete testing directory on host with possibility to set attributes:
+    - path
+    - mode
+    - selinux context
+    - access control list (facl)
+- port availability check
+- check selinux status of host
+- run a command on host
+- easy random string generation
+- probe
+
+# Docker example
+
+TBD: intro
+
+```python
+# TODO: use more complex example with assertions and stuff
```

Comment:
I vote for an example where we run web server inside container via `python3 -m http.server` and utilize more functions from conu. I can do it if needed.

Reply:
https://github.com/fedora-modularity/conu/pull/71